### ISSUE TITLE
Allow a Storage Bus to be marked as a 'infinite' supply of items

### ISSUE THREAD

**Issue by moonheart08** [AppliedEnergistics/Applied-Energistics-2#3268]
#### Description
This would make the system treat the items in the bus's filter as a infinite supply, allowing recipes to use that item even if there is only 1 and it needs 1 million. This does NOT mean item duplication; the items needed are pulled from the storage bus when avaliable, and anything using them otherwise will halt, just like normal behavior when a item runs out.
This would allow, say, a cobble gen to produce quintuple compressed cobblestone via AE2 without having all the required cobblestone ready yet.
the 'infinite' supply mode would either be a normal toggleable option or a modifier card, either one works. Infinite supply mode will NOT require that all the requested item be ready, it will steadily draw it in.

TL;DR Allow a item to pretend to be infinite even when it's not, so it's used once some is avaliable.

**Comment by yueh** [comment]
That is simply not possible as the whole network is desigend around a (known) finite amount which can be extracted in a single and instantaneous operation.

There are ways via the API to announced items as infinite supply, but that is for autocrafting only. AE2 itself supports this via level emitters e.g. to turn on a cobble gen when needed. Or otherwise left to addons/mod integrations adding different types.

**Comment by moonheart08** [comment]
Makes sense, i alrready assumed it would at least be 'next restructuring' if it even was accepted.

**Comment by HalfBlockStyle** [comment]
Now the only thing left to mention is what those means of supplying infinite crafting items actualy are:
ME-Level Emitter + Crafting Card
->set to "Crafting Emitter Mode"

Even if you don't use the redstone Signal to turn on your cobble gen, this reports what ever item you set it to to be infinitely available for crafting...
